### ISSUE TITLE
t/pod_snippets.t uses PadWalker.pm but it is not set as a prereq

### ISSUE THREAD

**Issue by szabgab** [pflanze/functional-perl#3]
I just cloned the repo and tried to run the tests. Got this:

```
$ make test
PERL_DL_NONLAZY=1 "/opt/perl-5.30.1/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
t/csv_to_xml.t ....................... ok   
t/dbi.t .............................. ok   
t/examples-fibs.t .................... skipped: could not use PadWalker
t/examples-functional-classes.t ...... ok   
t/examples-logic.t ................... skipped: Test requires module 'Function::Parameters' but it's not found
t/examples-primes.t .................. skipped: could not use Method::Signatures
t/functional_XML-t-div.t ............. ok   
t/functional_XML-test.t .............. ok   
t/htmlgen.t .......................... skipped: Test requires module 'Text::Markdown' but it's not found
t/intro-basics.t ..................... skipped: could not use FP::Repl::Dependencies
t/perl-goto-leak.t ................... ok   
t/perl-weaken-coderef-correctness.t .. ok   
t/perl-weaken-coderef.t .............. ok   
t/pod_snippets.t ..................... Can't locate PadWalker.pm in @INC (you may need to install the PadWalker module) (@INC contains: ./.htmlgen ./lib ./meta /home/gabor/cpan/functional-perl/blib/lib /home/gabor/cpan/functional-perl/blib/arch /home/gabor/perl5/lib/perl5//5.30.1/x86_64-linux /home/gabor/perl5/lib/perl5//5.30.1 /home/gabor/perl5/lib/perl5//x86_64-linux /home/gabor/perl5/lib/perl5/ /opt/perl-5.30.1/lib/site_perl/5.30.1/x86_64-linux /opt/perl-5.30.1/lib/site_perl/5.30.1 /opt/perl-5.30.1/lib/5.30.1/x86_64-linux /opt/perl-5.30.1/lib/5.30.1 .) at lib/FP/Repl/StackPlus.pm line 60.
 at lib/FP/Repl/StackPlus.pm line 60.
	FP::Repl::StackPlus::BEGIN() called at lib/FP/Repl/StackPlus.pm line 60
	eval {...} called at lib/FP/Repl/StackPlus.pm line 60
	require FP/Repl/StackPlus.pm called at lib/FP/Repl/Repl.pm line 125
	FP::Repl::Repl::BEGIN() called at lib/FP/Repl/StackPlus.pm line 60
	eval {...} called at lib/FP/Repl/StackPlus.pm line 60
	require FP/Repl/Repl.pm called at lib/FP/Repl.pm line 53
	FP::Repl::BEGIN() called at lib/FP/Repl/StackPlus.pm line 60
	eval {...} called at lib/FP/Repl/StackPlus.pm line 60
	require FP/Repl.pm called at lib/FP/Repl/WithRepl.pm line 109
	FP::Repl::WithRepl::BEGIN() called at lib/FP/Repl/StackPlus.pm line 60
	eval {...} called at lib/FP/Repl/StackPlus.pm line 60
	require FP/Repl/WithRepl.pm called at t/pod_snippets.t line 26
	main::BEGIN() called at lib/FP/Repl/StackPlus.pm line 60
	eval {...} called at lib/FP/Repl/StackPlus.pm line 60
BEGIN failed--compilation aborted at lib/FP/Repl/StackPlus.pm line 60.
 at lib/FP/Repl/StackPlus.pm line 60.
	require FP/Repl/StackPlus.pm called at lib/FP/Repl/Repl.pm line 125
	FP::Repl::Repl::BEGIN() called at lib/FP/Repl/StackPlus.pm line 60
	eval {...} called at lib/FP/Repl/StackPlus.pm line 60
	require FP/Repl/Repl.pm called at lib/FP/Repl.pm line 53
	FP::Repl::BEGIN() called at lib/FP/Repl/StackPlus.pm line 60
	eval {...} called at lib/FP/Repl/StackPlus.pm line 60
	require FP/Repl.pm called at lib/FP/Repl/WithRepl.pm line 109
	FP::Repl::WithRepl::BEGIN() called at lib/FP/Repl/StackPlus.pm line 60
	eval {...} called at lib/FP/Repl/StackPlus.pm line 60
	require FP/Repl/WithRepl.pm called at t/pod_snippets.t line 26
	main::BEGIN() called at lib/FP/Repl/StackPlus.pm line 60
	eval {...} called at lib/FP/Repl/StackPlus.pm line 60
Compilation failed in require at lib/FP/Repl/Repl.pm line 125.
 at lib/FP/Repl/Repl.pm line 125.
	FP::Repl::Repl::BEGIN() called at lib/FP/Repl/Repl.pm line 125
	eval {...} called at lib/FP/Repl/Repl.pm line 125
	require FP/Repl/Repl.pm called at lib/FP/Repl.pm line 53
	FP::Repl::BEGIN() called at lib/FP/Repl/Repl.pm line 125
	eval {...} called at lib/FP/Repl/Repl.pm line 125
	require FP/Repl.pm called at lib/FP/Repl/WithRepl.pm line 109
	FP::Repl::WithRepl::BEGIN() called at lib/FP/Repl/Repl.pm line 125
	eval {...} called at lib/FP/Repl/Repl.pm line 125
	require FP/Repl/WithRepl.pm called at t/pod_snippets.t line 26
	main::BEGIN() called at lib/FP/Repl/Repl.pm line 125
	eval {...} called at lib/FP/Repl/Repl.pm line 125
BEGIN failed--compilation aborted at lib/FP/Repl/Repl.pm line 125.
 at lib/FP/Repl/Repl.pm line 125.
	require FP/Repl/Repl.pm called at lib/FP/Repl.pm line 53
	FP::Repl::BEGIN() called at lib/FP/Repl/Repl.pm line 125
	eval {...} called at lib/FP/Repl/Repl.pm line 125
	require FP/Repl.pm called at lib/FP/Repl/WithRepl.pm line 109
	FP::Repl::WithRepl::BEGIN() called at lib/FP/Repl/Repl.pm line 125
	eval {...} called at lib/FP/Repl/Repl.pm line 125
	require FP/Repl/WithRepl.pm called at t/pod_snippets.t line 26
	main::BEGIN() called at lib/FP/Repl/Repl.pm line 125
	eval {...} called at lib/FP/Repl/Repl.pm line 125
Compilation failed in require at lib/FP/Repl.pm line 53.
 at lib/FP/Repl.pm line 53.
	FP::Repl::BEGIN() called at lib/FP/Repl.pm line 53
	eval {...} called at lib/FP/Repl.pm line 53
	require FP/Repl.pm called at lib/FP/Repl/WithRepl.pm line 109
	FP::Repl::WithRepl::BEGIN() called at lib/FP/Repl.pm line 53
	eval {...} called at lib/FP/Repl.pm line 53
	require FP/Repl/WithRepl.pm called at t/pod_snippets.t line 26
	main::BEGIN() called at lib/FP/Repl.pm line 53
	eval {...} called at lib/FP/Repl.pm line 53
BEGIN failed--compilation aborted at lib/FP/Repl.pm line 53.
 at lib/FP/Repl.pm line 53.
	require FP/Repl.pm called at lib/FP/Repl/WithRepl.pm line 109
	FP::Repl::WithRepl::BEGIN() called at lib/FP/Repl.pm line 53
	eval {...} called at lib/FP/Repl.pm line 53
	require FP/Repl/WithRepl.pm called at t/pod_snippets.t line 26
	main::BEGIN() called at lib/FP/Repl.pm line 53
	eval {...} called at lib/FP/Repl.pm line 53
Compilation failed in require at lib/FP/Repl/WithRepl.pm line 109.
 at lib/FP/Repl/WithRepl.pm line 109.
	FP::Repl::WithRepl::BEGIN() called at lib/FP/Repl/WithRepl.pm line 109
	eval {...} called at lib/FP/Repl/WithRepl.pm line 109
	require FP/Repl/WithRepl.pm called at t/pod_snippets.t line 26
	main::BEGIN() called at lib/FP/Repl/WithRepl.pm line 109
	eval {...} called at lib/FP/Repl/WithRepl.pm line 109
BEGIN failed--compilation aborted at lib/FP/Repl/WithRepl.pm line 109.
 at lib/FP/Repl/WithRepl.pm line 109.
	require FP/Repl/WithRepl.pm called at t/pod_snippets.t line 26
	main::BEGIN() called at lib/FP/Repl/WithRepl.pm line 109
	eval {...} called at lib/FP/Repl/WithRepl.pm line 109
Compilation failed in require at t/pod_snippets.t line 26.
 at t/pod_snippets.t line 26.
	main::BEGIN() called at t/pod_snippets.t line 26
	eval {...} called at t/pod_snippets.t line 26
BEGIN failed--compilation aborted at t/pod_snippets.t line 26.
 at t/pod_snippets.t line 26.
t/pod_snippets.t ..................... Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
t/predicates.t ....................... ok   
t/repl.t ............................. skipped: Test requires module 'FP::Repl::Dependencies' but it's not found
t/require_and_run_tests.t ............ not activating since running in test mode at lib/FP/Repl/AutoTrap.pm line 62.
t/require_and_run_tests.t ............ 1/2 Reference is already weak at lib/FP/Stream.pm line 757.
t/require_and_run_tests.t ............ ok   
t/skip-internal.t .................... ok   
t/skip-leak.t ........................ ok   
t/skip.t ............................. ok    
t/testlazy.t ......................... ok   
t/testlazy10.t ....................... ok   
t/trampoline-fix.t ................... skipped: Test requires module 'Method::Signatures' but it's not found
t/universal-isa.t .................... ok   

Test Summary Report
-------------------
t/pod_snippets.t                   (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: No plan found in TAP output
Files=24, Tests=52,  8 wallclock secs ( 0.10 usr  0.03 sys +  6.80 cusr  0.92 csys =  7.85 CPU)
Result: FAIL
Failed 1/24 test programs. 0/52 subtests failed.
make: *** [Makefile:1478: test_dynamic] Error 255
```

**Comment by pflanze** [comment]
Thanks for your report! The referenced commit should fix it (reopen in case it doesn't!), it will be in the next release.

**Comment by szabgab** [comment]
Great. This fix allowed me to send you #4 without errors.
